### ISSUE TITLE
Cutting and pasting an .fsx file in Solution Explorer errors

### ISSUE THREAD

**Issue by isaacabraham** [Microsoft/visualfsharp#732]
If you cut a file from one folder to another in VS2015, you'll get an "Object Reference not set to an instance" dialog. The paste operation works, but not the cut. In addition, the words "Copy of " will be inserted at the start of the filename of the pasted file.

**Comment by enricosada** [comment]
@isaacabraham i see the bug

can you add the lines of the fsproj where the files are defined?
I'll try fix it, but i want to test also your case (there are some permutations, like linked item, real directory, etc)

**Comment by isaacabraham** [comment]
Here's the ItemGroup section of the fsproj.

```xml
  <ItemGroup>
    <Compile Include="AssemblyInfo.fs" />
    <Compile Include="Program.fs" />
    <None Include="App.config" />
    <None Include="paket.references" />
    <None Include="Scripts\Consume.fsx" />
    <None Include="Scripts\load-references.fsx" />
    <None Include="Scripts\load-project.fsx" />
  </ItemGroup>
```

In my case, if I cut/paste Consume.fsx to the root level (or back again), it causes the error.

I've also realised that it does succeed in moving the file - but it leaves the old element there in the project file, pointing to a non-existing file.

**Comment by saul** [comment]
@KevinRansom Closed by #2692
